app/controllers/katello/api/v2/content_view_filters_controller.rb

```diff
@@ -12,21 +12,28 @@ class Api::V2::ContentViewFiltersController < Api::V2::ApiController
     param_group :search, Api::V2::ApiController
     param :content_view_id, :identifier, :desc => N_("content view identifier"), :required => true
     param :name, String, :desc => N_("filter content view filters by name")
+    param :types, Array, :desc => N_("types of filters")
     def index
       respond(:collection => scoped_search(index_relation.uniq, :name, :asc))
     end
 
     def index_relation
       query = ContentViewFilter.where(:content_view_id => (@view || ContentView.readable))
       query = query.where(:name => params[:name]) if params[:name]
+      if params[:types]
+        types = params[:types].each.collect do |type|
```

Comment:
When creating a new CV filter for Yum Content from the UI, this generates an error indicating:
```
2017-01-06T10:26:23 77a51cb7 [app] [E] NoMethodError: undefined method `each' for "rpm,package_group,erratum":String
 | /home/vagrant/katello/app/controllers/katello/api/v2/content_view_filters_controller.rb:24:in `index_relation'
...
```